collectd/files/postgresql.conf

```diff
@@ -6,16 +6,66 @@
 # Modify the config that generates this file instead
 #
 
-LoadPlugin postgresql
+<LoadPlugin postgresql>
```

Comment:
Are you sure this syntax is correct, because all other files seems to use the previous one

Reply:
in any case, for consistency, I think it should be better to stick to the previous syntax